docs/my-website/docs/observability/langtrace_integration.md

```diff
@@ -0,0 +1,43 @@
+import Image from '@theme/IdealImage';
+
+# 🔥 Langtrace AI
+Monitor, evaluate & improve your LLM apps
+
+## Pre-Requisites
+Make an account on [Langtrace AI](https://langtrace.ai/login)
+
```

Comment:
Add a link / example of how to use with LiteLLM proxy - majority of our users use the proxy @alizenhom